### PR TITLE
create telemetry event enum and `logUsage()` wrapper

### LINKED ISSUE #489: Set up enum for telemetry events
We currently use `getTelemetryLogger().logUsage()` but could set up a convenience function and enum like we do for setting context values (see [`setContextValue()`](https://github.com/confluentinc/vscode/blob/6f301aae2a91c314697e2d9659696f793c11f9e5/src/context.ts#L18) and [`ContextValues`](https://github.com/confluentinc/vscode/blob/6f301aae2a91c314697e2d9659696f793c11f9e5/src/context.ts#L36)). This will help limit the number of unique events we send, as well as add comments for context and helping future contributors determine whether to reuse an existing event or create a new one.

### PR DESCRIPTION
**Pull request by shouples**
## Summary of Changes

Closes #489.

Migrates our use of `getTelemetryLogger().logUsage(<string>, data)` to use an enum, which should help us more quickly determine what events are being sent, where we might want/need to clean up or standardize event names, etc. Majority of the changes are in here: https://github.com/confluentinc/vscode/blob/6905985c65970621d59271ffc40174f7f962eb2b/src/telemetry/events.ts#L5-L36

And the rest of the changes are updating imports and previous `getTelemetryLogger().logUsage(<string> ...)` calls.

Same general pattern used in [`setContextValue()`](https://github.com/confluentinc/vscode/blob/main/src/context/values.ts#L12).

## Pull request checklist

Please check if your PR fulfills the following (if applicable):

##### Tests

- [ ] Added new
- [ ] Updated existing
- [ ] Deleted existing

##### Other

- [ ] All new disposables (event listeners, views, channels, etc.) collected as  for eventual cleanup?

- [ ] Does anything in this PR need to be mentioned in the user-facing [CHANGELOG](https://github.com/confluentinc/vscode/blob/main/CHANGELOG.md) or [README](https://github.com/confluentinc/vscode/blob/main/public/README.md)?
- [ ] Have you validated this change locally by [packaging](https://github.com/confluentinc/vscode/blob/main/README.md#packaging-steps) and installing the extension `.vsix` file?
  ```shell
  gulp clicktest
  ```

### CODE REVIEW

File: src/docker/workflows/confluent-local.ts [hunk 1]
```diff
@@ -101,7 +102,7 @@ export class ConfluentLocalWorkflow extends LocalResourceWorkflow {
     }
     count = parseInt(numContainersString, 10);
     const plural = count > 1 ? "s" : "";
-    this.sendTelemetryEvent("Input Box Filled", {
+    this.sendTelemetryEvent(UserEvent.InputBoxFilled, {
```
Comment (noeldevelops) [inline src/docker/workflows/confluent-local.ts:105]: Not related to this PR but why do we collect this event? Seems random.

**Review by noeldevelops — APPROVED** [review]
Tested the branch locally with the Segment Dev env key and I can see all the events coming through as expected. Code looks good and I think will be really helpful as we get more strategic about telemetry. Thanks! 

We talked about this, but for the record here - in the future we may consider combining all the `scaffold` and `docker` event types to reduce unique names (due to the constraint on unique event types in final data destinations).

File: src/docker/workflows/confluent-local.ts (hunk 1 above)
Comment (shouples) [reply src/docker/workflows/confluent-local.ts:105]: Originally it was just to gauge when someone filled out the "how many brokers/containers" versus exiting early, but I'm good with removing it entirely if we want

Comment (noeldevelops) [reply src/docker/workflows/confluent-local.ts:105]: oh that does seem useful; makes sense

**Pull request merged**